ads/google/a4a/utils.js

```diff
@@ -293,3 +310,83 @@ export function getCorrelator(win, opt_cid) {
   }
   return win.ampAdPageCorrelator;
 }
+
+/**
+ * Gets a relatively terse and controlled-format string version of the user
+ * agent.
+ *
+ * @param {!Window} win
+ * @return {string}
+ */
+export function userAgentString(win) {
```

Comment:
Why do you need to include this on the CSI ping?  We already record UserAgent as part of gen_204s